File: C#/BluffinMuffin.Protocol/Lobby/QuickMode/IdentifyResponse.cs
using Com.Ericmas001.Net.Protocol;

namespace BluffinMuffin.Protocol.Lobby.QuickMode
{
    /// <summary>
    /// 
    /// </summary>
    public class IdentifyResponse : AbstractBluffinReponse<IdentifyCommand>
    {
        /// <summary>
        /// Indicates if the identification was successful
        /// </summary>
        [ExampleValue(true)]
        public bool Ok { get; set; }
        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public IdentifyResponse(IdentifyCommand command)
            : base(command)
        {
        }
    }
}

File: Python/bluffin_muffin/protocol/lobby/quick_mode/identify_response.py
from Python.bluffin_muffin.protocol.abstract_response import AbstractResponse
from identify_command import IdentifyCommand

__author__ = 'ericmas001@gmail.com'



class IdentifyResponse(AbstractResponse):
    def __init__(self, obj):
        super().__init__(obj, IdentifyCommand(obj['Command']))